README.md

```diff
@@ -24,7 +24,12 @@ The goal of this document is to serve as a good starting point for someone onboa
 5. Clone the repository and open the pisp\_app from android studio
 6. Go to settings -> flutter and add the path to flutter SDK if not already present.
 7. Get your google-services.json from firebase. Follow the steps [here](https://www.digitalocean.com/community/tutorials/flutter-firebase-setup) This is necessary to build the app successfully.
-8. Connect device/ emulator and run the app.
+8. Make sure you enter the app's SHA-1 fingerprint under the [Settings page](https://console.firebase.google.com/u/0/project/_/settings/general) of your Firebase console as this is required to ensure that Google sign-in functions properly. You can get the app's debug SHA-1 fingerprint by running this command: `keytool -list -v -alias androiddebugkey -keystore %USERPROFILE%\.android\debug.keystore`.
```

Comment:
Would it be better to give a note in the documentation that we currently do not support the iOS version?

Reply:
I added a note